react-redux/react-redux.d.ts

```diff
@@ -54,16 +54,16 @@ declare module "react-redux" {
     pure: boolean;
   }
 
-  export interface Property {
+  export interface Property<S> {
     /**
      * The single Redux store in your application.
      */
-    store?: Store;
+    store?: Store<S>;
     children?: Function;
   }
 
   /**
    * Makes the Redux store available to the connect() calls in the component hierarchy below.
    */
-  export class Provider extends Component<Property, {}> { }
+  export class Provider extends Component<Property<any>, {}> { }
```

Comment:
Is this change compatible with [react-redux@4.0.0](https://github.com/rackt/react-redux/releases/tag/v4.0.0)

```
<Provider> no longer accepts a function child
```
If yes, can we set the version number at the first line to 4.0.0 ?